process/sync/shardblock_test.go

```diff
@@ -930,8 +930,8 @@ func TestBootstrap_SyncBlockShouldCallForkChoice(t *testing.T) {
 	hasher := &mock.HasherMock{}
 	marshalizer := &mock.MarshalizerMock{}
 	forkDetector := &mock.ForkDetectorMock{}
-	forkDetector.CheckForkCalled = func() (bool, uint64) {
-		return true, math.MaxUint64
+	forkDetector.CheckForkCalled = func() (bool, uint64, []byte) {
+		return true, math.MaxUint64, nil
```

Comment:
add test for this special scenario as discussed.

Reply:
Done